src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.math.Pair;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Calibrations;
+import frc.robot.Constants;
+import frc.robot.RobotMap;
+
+import java.util.TreeMap;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkMaxPIDController;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+
+public class Lift extends SubsystemBase {
```

Comment:
Can we add a comment for the whole lift that says it contains the elevator, arm, and game piece grabber

Reply:
ok